processor/deltatocumulativeprocessor/internal/data/expo_test.go

```diff
@@ -37,6 +38,30 @@ func TestExpoAdd(t *testing.T) {
 		dp:   expdp{PosNeg: bins{0, 0, 0, 0, 0, 0, 0, 0}.Into(), Count: 0},
 		in:   expdp{PosNeg: bins{1, 2, 3, 4, 5, 6, 7, 8}.Into(), Count: 2 * (1 + 2 + 3 + 4 + 5 + 6 + 7 + 8)},
 		want: expdp{PosNeg: bins{1, 2, 3, 4, 5, 6, 7, 8}.Into(), Count: 2 * (0 + (1 + 2 + 3 + 4 + 5 + 6 + 7 + 8))},
+	}, {}, {
+		name:    "maxsize/1",
+		dp:      expdp{PosNeg: bins{0, 0, 0, ø}.Into(), Count: 0},
+		in:      expdp{PosNeg: bins{ø, ø, ø, ø, 1, 2, 3, 4}.Into(), Count: 2 * (1 + 2 + 3 + 4)},
+		want:    expdp{PosNeg: bins{ø, ø, 0, 10, ø}.Into(), Scale: -3, Count: 2 * (0 + (1 + 2 + 3 + 4))},
```

Comment:
I do not understand this test case.

The max length is 1, but `want` specifies 2 buckets (0, 10). why?

Reply:
Good question. When downscaling, there is an [edge case](https://github.com/edma2/opentelemetry-collector-contrib/blob/022c83f96966a4ab3a9e5fd5acf472d3f3821ecf/processor/deltatocumulativeprocessor/internal/data/expo/scale.go#L76) where we may need to extend the bucket by 1 in order to fit all the counts. So the actual bucket size might be 1 larger than max scale.